### PR TITLE
Khwe 23 render songs for the artist

### PR DESCRIPTION
**Pull request by WOLFRIEND**
image

### CODE REVIEW

File: src/components/Artists/Artists.html [hunk 1]
```diff
@@ -2,30 +2,30 @@
...
-          class="mdc-card mdc-card__primary-action mdc-ripple artists__ripple"
-        >
-          <div class="mdc-card__media mdc-card__media--square artists__media">
-            <img
-              class="artists__image"
-              src="<%= item.imageURL %>"
-              alt="artist image"
-            />
+      <a class="artists__link main__list-item" href="/artists/<%= item.id %>">
+        <div class="artists__card_shaped">
```
Comment (ihor4568) [inline src/components/Artists/Artists.html]: modifier can not be used without an element or block

File: src/components/Artists/Artists.html [hunk 2]
```diff
@@ -2,30 +2,30 @@
...
-        <div
-          class="mdc-card mdc-card__primary-action mdc-ripple artists__ripple"
-        >
-          <div class="mdc-card__media mdc-card__media--square artists__media">
-            <img
-              class="artists__image"
-              src="<%= item.imageURL %>"
-              alt="artist image"
-            />
+      <a class="artists__link main__list-item" href="/artists/<%= item.id %>">
```
Comment (ihor4568) [inline src/components/Artists/Artists.html]: main__list-item ?? how can it be here? it is an element of the main block

File: src/components/ArtistSongTable/ArtistSongTable.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    flex-wrap: wrap;
+    margin-top: 1rem;
+  }
+
+  &__card {
+    box-shadow: none;
+    width: 100%;
+  }
+
+  &__card_shaped {
```
Comment (ihor4568) [inline src/components/ArtistSongTable/ArtistSongTable.scss]: you can use nesting

File: src/components/ArtistSongTable/ArtistSongTable.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      })
+      .then(songsInfo => {
+        songsInfo.forEach((item, i) => {
+          const [album, authorsInfo] = item;
+
+          this.songs[i].album = album;
+          this.songs[i].authorsInfo = authorsInfo;
+        });
+
+        this.mount(false);
```
Comment (ihor4568) [inline src/components/ArtistSongTable/ArtistSongTable.js]: you can pass nothing here

File: src/components/ArtistSongTable/ArtistSongTable.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      <div class="mdc-card artist-song-table__card responsive-grid">
+        <a class="artist-song-table__link main__list-item" href="">
+          <div class="artist-song-table__card_shaped">
+            <div
+              class="mdc-card mdc-card__primary-action mdc-ripple artist-song-table__ripple"
+            >
+              <div
+                class="mdc-card__media mdc-card__media--square artist-song-table__media"
+              >
+                <% if( imageUrl ) { %>
```
Comment (ihor4568) [inline src/components/ArtistSongTable/ArtistSongTable.html]: if (imageUrl) {

please format this js line

File: src/components/ArtistSongTable/ArtistSongTable.scss [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    flex-wrap: wrap;
+    margin-top: 1rem;
+  }
+
+  &__card {
+    box-shadow: none;
+    width: 100%;
+  }
+
+  &__artist-card_shaped {
```
Comment (ihor4568) [inline src/components/ArtistSongTable/ArtistSongTable.scss]: where is the element for this modifier?

File: src/components/Main/Main.js [hunk 7]
```diff
@@ -253,7 +265,14 @@ export class MainComponent {
 
     this.notFound = new NotFoundComponent(this.mainContentPoint);
 
-    this.artist = new ArtistsComponent(this.mainContentPoint);
+    this.artist = new ArtistsComponent(this.mainContentPoint, {
+      onArtistClick: this.handleArtistView.bind(this)
```
Comment (ihor4568) [inline src/components/Main/Main.js]: handleArtistClick

**Review by OlegShynkarenko — APPROVED** [review]

**Pull request merged**